react/src/lib/components/DeckGLMap/components/settings/DrawModeSelector.test.tsx

```diff
@@ -6,9 +6,10 @@ import { testStore, Wrapper } from "../../test/TestWrapper";
 import DrawModeSelector from "./DrawModeSelector";
 
 describe("Test draw-mode menu", () => {
+    // The render function is rendering just 'null'
     it("snapshot test", () => {
         const { container } = render(
-            Wrapper({ children: <DrawModeSelector layerId="colormap-layer" /> })
+            Wrapper({ children: <DrawModeSelector layerId="" /> })
```

Comment:
The `layerId` prop needs to match the DrawingLayer `id` field from deckgl-map.json. Now it is `"drawing-layer"`.
Same for the rest of DrawModeSelector components in this file.

Reply:
Thank you so much @alexandruandrei91 for solving all these issues :)
The snapshots are all fine now.